src/gui/torrentcontentmodelitem.h

```diff
@@ -32,17 +32,7 @@
 #include <QList>
 #include <QVariant>
 
-namespace prio
-{
-    enum FilePriority
-    {
-        IGNORED=0,
-        NORMAL=1,
-        HIGH=6,
-        MAXIMUM=7,
-        MIXED=-1
-    };
-}
+#include "base/bittorrent/torrenthandle.h"
```

Comment:
Seems like **FilePriority** should be declared in its own header...

Reply:
Do we want a new header file just for this one enum class? Seems it might be overkill, but I don't feel strongly.

Reply:
Overkill is include entire "torrenthandle.h" when we need only FilePriority class. @Chocobo1, what do you say? If it's no problem feel free to merge it.

Reply:
This idea actually has popped into my mind, I think there should be a file that has all these similar enum/struct defines (and their helper functions). Maybe "bitorrent/enums.{cpp, h}"?

Reply:
>I think there should be a file that has all these similar enum/struct defines (and their helper functions). Maybe "bitorrent/enums.{cpp, h}"?

I would not combine different types in one header (unless they are very likely used adjacently).

Reply:
I'll move the enum class to its own header file.

Reply:
Move is complete